### PR TITLE
Feat/clearance level trail

### PR DESCRIPTION
**Pull request by underdarknl**
### Changes
Moves the clearance level inspect feature for inherited clearance levels to the clearance level tab, and also fixes the layout there to be more sane when it comes to explaining the various types.

### Demo
image

---

### Code Checklist
- [x] All the commits in this PR are properly PGP-signed and verified;
- [x] This PR only contains functionality relevant to the issue; tickets have been created for newly discovered issues.
- [ ] I have written unit tests for the changes or fixes I made.
- [ ] For any non-trivial functionality, I have added integration and/or end-to-end tests.
- [ ] I have performed a self-review of my code and refactored it to the best of my abilities.

### Communication
- [ ] I have informed others of any required `.env` changes files if required and changed the `.env-dist` accordingly.
- [ ] I have made corresponding changes to the documentation, if necessary.

---
## Checklist for code reviewers:
Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_code.md) into your comment.

---
## Checklist for QA:
Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_qa.md) into your comment.

### CODE REVIEW

File: rocky/rocky/templates/scan_profiles/scan_profile_detail.html [hunk 1]
```diff
@@ -33,20 +31,22 @@ <h2>{% translate "Declared" %}</h2>
...
+                    {% include "partials/explanations.html" %}
+
                 {% endif %}
                 {% if user.acknowledged_clearance_level > -1 and organization_indemnification and user.trusted_clearance_level > -1 %}
-                    <form novalidate
-                          action="{% ooi_url "scan_profile_detail" ooi.primary_key organization.code %}"
+                    <h2>{% translate "Set clearance level" %}:</h2>
+                    <form id="set_clearance_level"
+                          novalidate
+                          action="{% ooi_url "scan_profile_detail" ooi.primary_key organization.code %}#set_clearance_level"
```
Comment (ammar92) [inline rocky/rocky/templates/scan_profiles/scan_profile_detail.html:49]: The anchor is indeed a nice addition

**Review by ammar92 — APPROVED** [review]
Looks good to me 👍 Just to be sure: for inherited scan levels the selection box should be preselected to "Please select one of the available options"?

**Comment by ammar92** [comment]
## Checklist for QA:
- [ ] I have checked out this branch, and successfully ran a fresh `make reset`.
- [x] I confirmed that there are no unintended functional regressions in this branch:
    - [ ] I have managed to pass the onboarding flow
    - [x] Objects and Findings are created properly
    - [x] Tasks are created and completed properly
- [x] I confirmed that the PR's advertised `feature` or `hotfix` works as intended.

### What works:
* Works as intended

**Comment by underdarknl** [comment]
> Looks good to me 👍 Just to be sure: for inherited scan levels the selection box should be preselected to "Please select one of the available options"?

I left that untouched. I'm also not sure why the form has novalidate on there, as there's no real change when we remove it.

**Review by dekkers — APPROVED** [review]

**Pull request merged**